renovate.json

```diff
@@ -18,6 +18,11 @@
       ],
       "groupName": "opentelemetry-collector upstream dependencies",
       "labels": ["upgrade-otelcol-libs"]
+    },
+    {
+      "matchDatasources": ["golang-version"],
+      "matchFileNames": ["go.mod"],
```

Comment:
Just to confirm, this PR aims to update only "go.mod" in the root directory and not update files under subdirectories, right?

Reply:
thanks. understand